tests/wasm/optional_primitives.js

```diff
@@ -102,6 +104,22 @@ exports.js_works = () => {
     assert.strictEqual(wasm.optional_u64_identity(wasm.optional_u64_min()), BigInt('0'));
     assert.strictEqual(wasm.optional_u64_identity(wasm.optional_u64_max()), BigInt('18446744073709551615'));
 
+    assert.strictEqual(wasm.optional_i128_identity(wasm.optional_i128_none()), undefined);
+    assert.strictEqual(wasm.optional_i128_identity(wasm.optional_i128_zero()), BigInt('0'));
+    assert.strictEqual(wasm.optional_i128_identity(wasm.optional_i128_one()), BigInt('1'));
+    assert.strictEqual(wasm.optional_i128_identity(wasm.optional_i128_neg_one()), BigInt('-1'));
+    assert.strictEqual(wasm.optional_i128_identity(wasm.optional_i128_min()), -170141183460469231731687303715884105728n);
+    assert.strictEqual(wasm.optional_i128_identity(wasm.optional_i128_max()), 170141183460469231731687303715884105727n);
+    assert.strictEqual(wasm.optional_i128_identity(0x3_1415_9265_3598_7932_3846n), 0x3_1415_9265_3598_7932_3846n);
+    assert.strictEqual(wasm.optional_i128_identity(-0x3_1415_9265_3598_7932_3846n), -0x3_1415_9265_3598_7932_3846n);
+
+    assert.strictEqual(wasm.optional_u128_identity(wasm.optional_u128_none()), undefined);
+    assert.strictEqual(wasm.optional_u128_identity(wasm.optional_u128_zero()), BigInt('0'));
+    assert.strictEqual(wasm.optional_u128_identity(wasm.optional_u128_one()), BigInt('1'));
+    assert.strictEqual(wasm.optional_u128_identity(wasm.optional_u128_min()), BigInt('0'));
+    assert.strictEqual(wasm.optional_u128_identity(wasm.optional_u128_max()), 340282366920938463463374607431768211455n);
```

Comment:
```suggestion
    assert.strictEqual(wasm.optional_u128_identity(wasm.optional_u128_max()), 340282366920938463463374607431768211455n);
```
Can't we just use hexadecimal representation here and in other similar ones as well?

Reply:
Good point.Hex would be a bit easier to understand here.